pkg/registry/apps/daemonset/strategy_test.go

```diff
@@ -138,11 +139,10 @@ func newDaemonSetWithSelectorLabels(selectorLabels map[string]string, templateGe
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: selectorLabels,
 				},
-				Spec: api.PodSpec{
-					RestartPolicy: api.RestartPolicyAlways,
-					DNSPolicy:     api.DNSClusterFirst,
-					Containers:    []api.Container{{Name: fakeImageName, Image: fakeImage, ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: api.TerminationMessageReadFile}},
-				},
+				Spec: podtest.MakePod("",
```

Comment:
Emphasizing comments before - none of this is relevant to the testcase, so it could all be reduced to :+1: 

```
Spec: podtest.MakePod("").Spec
```

In particular, the use of "fakeImageName" makes it LOOK like that's an important attribute of this test, but it's a red herring.